src/Ninja/G1CP/Content/Fixes/Gamesave/fix0051_StonehengeCryptChest.d

```diff
@@ -1,29 +1,30 @@
 /*
  * #51 Chest in stonehenge crypt behind wall
  */
+
+/*
+ * Make the positions available to the functions below
+ */
+const float G1CP_0051_StonehengeCryptChest_OriginalPos[3] = {-35498.5625, 2331.01416, -13830.9131};
+const float G1CP_0051_StonehengeCryptChest_CorrectedPos[3] = {-35311.8100, 2281.0100, -14096.8200};
```

Comment:
Since we change these anyway, "PosOld" and "PosNew" would result in shorter names (as done in #50). What do you think?

Reply:
Okay for me. I like consistency.